docs/content/posts/2024-08-15-release-cdc-3.2.0.md

```diff
@@ -0,0 +1,121 @@
+---
+title:  "Apache Flink CDC 3.2.0 Release Announcement"
+date: "2024-08-15T08:00:00.000Z"
+authors:
+- renqs:
+  name: "Qingsheng Ren"
+  twitter: "renqstuite"
+aliases:
+- /news/2024/08/15/release-release-cdc-3.2.0.html
+---
+
+The Apache Flink Community is excited to announce the release of Flink CDC 3.2.0!
+This release aims to improve usability and stability of existing features, 
+including transform and schema evolution. 
+Also, backwards-compatible code and tests have been added to help users upgrade from previous CDC versions more smoothly.
+
+Flink CDC release packages are available at [Releases Page](https://flink.apache.org/downloads.html#flink-cdc),
+and documentations are available at [Flink CDC documentation](https://nightlies.apache.org/flink/flink-cdc-docs-release-3.2) page.
+Looking forward to any feedback from the community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or [JIRA](https://issues.apache.org/jira/browse/flink)!
+
+# Highlights
+
+## Connectors
+
+### New Pipeline Connectors
+
+Flink CDC 3.2.0 introduces 1 new pipeline connector:
+
+* Elasticsearch sink (verified on Elasticsearch 6.8, 7.10, and 8.12)
+
+### Apache Paimon
+
+Paimon connector has bumped its dependency to an up-to-date version and could benefit from all bug fixes and improvements. 
+Also, there are other improvements including:
+
+* Added dynamic bucket mode support for data distribution cases.
+* Handle specified column positions correctly.
+* Fixed duplicate commit issue after failover.
+
+### Apache Kafka
+
+Now, it is possible to configure Kafka partitioning strategy with new `key.format` and `partition.strategy` configurations.
+
+### MySQL
+
+* `AlterColumnTypeEvent` could be correctly parsed from `MODIFY COLUMN` DDL. 
+* Assigning any column as snapshot chunk column is supported in both pipeline and source connector.
+* Fixed a bug that may cause infinite hanging with timestamp startup mode
+
+### MongoDB
+
+* MongoDB CDC connector has been tested and verified on MongoDB 7.x.
+
+## Kubernetes Deployment Mode
+
+Now it is possible to deploy a YAML pipeline job to Kubernetes cluster with CDC CLI. 
+See [Deployment Docs](https://nightlies.apache.org/flink/flink-cdc-docs-release-3.2/docs/deployment/kubernetes/) for more details.
+
+## Customizable Schema Evolution
+
+More customizable schema evolution modes have been added.
+Now, apart from existing "Evolve", "Ignore", and "Exception" mode, a pipeline job could be configured as:
+
+* "TryEvolve": Always do schema evolution attempts, but evolution failures will be tolerated.
+* "Lenient": Similar to "TryEvolve" mode, but provides more lenient schema conversion after failover.
```

Comment:
Could you give some example to explain the lenient behavior? Eg. what will happen when ``drop column`,`add column` event meets  lenient behavior.